tests/methods/gasci-dsrg-mrpt2-2/input.dat

```diff
@@ -0,0 +1,52 @@
+import forte
+memory 2gb
+
+molecule co{
+C     0.000000     0.000000     0.70
+O     0.000000     0.000000    -0.70
+}
+
+set {
+  scf_type            df
+  reference           rohf
+  e_convergence       8
+  d_convergence       8
+  basis               cc-pvdz
+  df_basis_mp2        cc-pvdz-jkfit
+  ints_tolerance      0.0
+}
+Ezero, wfn = energy('scf', return_wfn=True)
+
+set forte {
+  job_type              newdriver
+  int_type              df
+  active_space_solver   detci
+  correlation_solver    sa-mrdsrg
+  dsrg_s                0.0
+  calc_type             sa
+  active_ref_type       gas
+  avg_state             [[0,1,1],[2,1,1],[3,1,1]]
+  avg_weight            [[0],[1],[1]]
+  gas1                  [1,0,0,0]
+  gas2                  [5,0,2,2]
+  gas1max               [2,1,1]
+  gas1min               [2,1,1]
+  transition_dipoles    true
+  e_convergence         8
+  r_convergence         7
+}
+edsrg = energy('forte', ref_wfn = wfn)
+
+#psi4.core.print_variables()
+
+dip_a1 = variable("SINGLET <0A1|DM_Z|0A1> ENTRY 0")
+dip_b1 = variable("SINGLET <0B1|DM_Z|0B1> ENTRY 0")
+dip_b2 = variable("SINGLET <0B2|DM_Z|0B2> ENTRY 0")
+compare_values(dip_a1, variable("SINGLET <0A1|DM_Z|0A1> ENTRY 1"), 8, "Dipole Z singlet A1 root 0")
```

Comment:
This does not look like a comparison with a reference number. Can you please compare to a numerical value instead so that we can detect issues with the code in the future?

Reply:
This is actually the intended test that the dipole moments should not be changed before and after DSRG with s = 0 due to orbital changes.

Reply:
Ok, but you can imagine cases under which this test will pass but the dipole is wrong for other reasons. I would then add a comment to the test to explain what you are testing.